tests/App/Entity/Context.php

```diff
@@ -33,6 +33,6 @@ class Context extends BaseContext
 
     public function getId()
     {
-        return $this->id;
+        return $this->id ?? '';
```

Comment:
Isn't better to return null if there is no id and check for null value if needed ?

Reply:
Yep, but that change needs to be done on SonataClassificationBundle.

Reply:
Why ? What is the error you try to fix ?

Reply:
This is the Context entity that is created in order to test the integration with classification-bundle

(Context entity comes from classification-bundle)

Reply:
I've made this https://github.com/sonata-project/SonataClassificationBundle/pull/689

Reply:
It's merged @jordisala1991

Reply:
Fixed.